tools/Raven.Debug/CommandLineApp.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Diagnostics.Runtime;
+using Microsoft.Diagnostics.Runtime.Interop;
+using Microsoft.Extensions.CommandLineUtils;
+using Newtonsoft.Json;
+
+namespace Raven.Debug
+{
+    internal class CommandLineApp
+    {
+        private const string HelpOptionString = "-h | -? | --help";
+
+        private static CommandLineApplication _app;
+
+        private const int LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR = 0x00000100;
+
+        [DllImport("kernel32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
+        private static extern IntPtr LoadLibraryEx(string lpFileName, IntPtr hReservedNull, int dwFlags);
+
+        private static void EnsureProperDebugDllsAreLoaded()
+        {
+            var systemDirectory = Environment.GetFolderPath(Environment.SpecialFolder.System);
+
+            var res = LoadLibraryEx(Path.Combine(systemDirectory, "dbghelp.dll"), IntPtr.Zero, LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR);
+            if (res == IntPtr.Zero)
+                throw new Win32Exception(Marshal.GetLastWin32Error());
```

Comment:
Need to explain here that we couldn't find `dbghelp.dll` and how to get ti

Reply:
It was fixed by us (you) in 3.5. We are taking the DLL from the system dir, and the files comes with the OS so it needs to be there.